src/sentry/snuba/dataset.py

```diff
@@ -82,3 +82,4 @@ class EntityKey(Enum):
 @unique
 class StorageKey(Enum):
     ProfileChunks = "profile_chunks"
+    SearchIssues = "search_issues"
```

Comment:
`search_issues` is already defined in Dataset and EntityKey. Do you know if we need it also defined as a storage key?